src/annotations/ProvidesToListAnnotation.ts

```diff
@@ -0,0 +1,21 @@
+import { AnnotationKey, Target } from '../annotations/Annotations';
+import { BindingAnnotation } from '../annotations/BindingAnnotation';
+import { ProvidesAnnotation } from '../annotations/ProvidesAnnotation';
+import { Binder, ListBinder } from '../binder';
+
+export class ProvidesToListAnnotation extends BindingAnnotation {
+  static _name = 'ProvidesToListAnnotation';
+
+  private readonly _sortOrder?: number;
```

Comment:
perhaps should make this an enum or a narrower type than number.

Reply:
I think this needs to be as general as _number_. If it needs a change, I think it's better to change it when we see a need for it.

Reply:
Sort order for me seems like it should only have a few possible values and we should capture the possibility of errors if we can no? Like what does a sort order of 69 or 420 or 13.37 mean?

Reply:
Yeah, but then we introduce restrictions that we currently don't have. I don't think it should change for _this PR_, if it should change at all.